src/Microsoft.Azure.WebJobs/ParameterBindingData.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// A reference type for supporting SDK-type bindings in out-of-process Function workers
+    /// </summary>
+    public class ParameterBindingData
+    {
+        /// <summary> Initializes a new instance of the <see cref="ParameterBindingData"/> class with the Content-Type set to application/json</summary>
```

Comment:
nit: looks like you're adding an unnecessary space at the beginning of each of these lines.